src/components/experiments/single-view/results/MetricAssignmentResults.tsx

```diff
@@ -399,6 +399,10 @@ export default function MetricAssignmentResults({
   const isVariationSelected = (variation: Variation) =>
     [baseVariationId, changeVariationId].includes(variation.variationId)
 
+  const variationsTuples = isMultivariation
+    ? cartesianProduct(experiment.variations, experiment.variations)
+    : [variations.slice().reverse()]
```

Comment:
Yeah this is a good solution.

```suggestion
    // Need to reverse the A/B variations for the grid (e.g. Treatment - Control):
    : [[variations[1], variations[0]]]
```
I think this would make what we are doing clearer.

Reply:
Yes, 100%! Otherwise, it would require a bit of code reading in order to get even why we're using the locally defined `variations` Array.